### ISSUE TITLE
Cleanup post-add-browserify

### ISSUE THREAD

**Issue by viddo** [CartoDB/cartodb.js#722]
:fire: to take place after #699:, to be removed once verified not actually needed:
- [x] concat task, nojquery/noleaflet no long needed (already done in 6ae2335b3adccaf96ceb5155033779fc657b9262)
- [x] `index.js` old rails manifest?
- [x] `src-browserify/loader.js`
- [x] `geo/ui/switcher*`
- [x] `test/lib`, `sinon` no longer used in new specs
- [x] Rename `_Promise` to `Promise`
- [x] Review files/classes that declara plain Backbone.Model and Backbone.Collection (eg: [geo/geometries.js](https://github.com/CartoDB/cartodb.js/blob/v4-browserify-standard2/src-browserify/geo/geometries.js))
- [x] `package.json` browser/browserify-shim definitions, all used?
- [x] `Makefile`, really necessary nowadays? 
- [x] `scripts` dir, e.g. `scripts/compress`, only be used for cartodbeditor; can be removed/moved?
- [x] `grunt/`, e.g. templates used in core concat, no longer needed (e.g. underscore_no_conflict, version_header etc.).
- [x] `vendor/reqwest` (unused since not supporting core or nojquery bundles anymore)
- [x] `vendor/mod/torque.uncompressed`, now required through torque.js npm bundle?

Out-of-scope/postponed:
- ~~rm `test/demos/`~~ :point_right: #715
- ~~`src/` + `test/spec` (but not `src-browserify`)~~ :point_right: #758
- CartoDB editor also will need to be updated at some point :point_right: cartodb/cartodb#6068

**Comment by viddo** [comment]
closed in #763
